src/screens/Profile/Login/ConfirmCode.tsx

```diff
@@ -64,6 +69,20 @@ export default function ConfirmCode({navigation, route}: ConfirmCodeProps) {
     }
   };
 
+  // User might be automatically logged in with Firebase auth, but only on Android
+  // Check user from state and see if it is updated while we wait
+  useEffect(() => {
+    if (user?.phoneNumber) {
+      // OK To use toast here, as this is Android only for now
+      ToastAndroid.showWithGravity(
```

Comment:
I am unfamiliar with this ToastAndroid. Will look into this, and it also should be run by the designers. Do you have a screenshot?

Reply:
Toasts are an OS-level feature on Android (and thus styled by the OS), they are non-interactive and disappear after a short while. See the first paragraph here: https://developer.android.com/guide/topics/ui/notifiers/toasts

Screen shot from the app:
![Screenshot_20210318-164504_atb](https://user-images.githubusercontent.com/4928324/111657840-1a782a80-880c-11eb-9e81-1bb2c8de34c4.jpg)

Reply:
We do not rely on Toasts anywhere else, so I'd like to run this by design CC @pettersuul. Is there some other way we could better notify the user that the login was successful if the SMS-code was retrieved automatically?

Reply:
@pettersuul did you have a look at this?

Reply:
I'm not sure a confirmation is necessary, as the user is informed if the login is unsuccessful, and will immediately see the logged in phone nummer if successful

Reply:
OK, I'll remove the toast.